doc/tutorials/dnn/dnn_yolo/dnn_yolo.markdown

```diff
@@ -0,0 +1,52 @@
+YOLO DNNs  {#tutorial_dnn_yolo}
+===========================
+
+Introduction
+------------
+
+In this text you will learn how to use opencv_dnn module using yolo_object_detection (Sample of using OpenCV dnn module in real time with device capture, video and image).
+
+We will demonstrate results of this example on the following picture.
+![Picture example](images/yolo.jpg)
+
+Examples
+-----------
```

Comment:
Please align "header" marker (`-----`) to the text above (like lines 4-5).
Same note for line 2.